test/AddressRegistry.Tests/AggregateTests/WhenReaddressing/GivenDestinationAddressDoesNotExist.cs

```diff
@@ -190,10 +190,6 @@ public void WithSourceAddressHasBoxNumbers_ThenDestinationAddressHasBoxNumbers()
                             proposedBoxNumberAddress.GeometryMethod,
                             proposedBoxNumberAddress.GeometrySpecification,
                             new ExtendedWkbGeometry(proposedBoxNumberAddress.ExtendedWkbGeometry))),
-                    new Fact(_streamId,
-                        new AddressWasRejected(
-                            _streetNamePersistentLocalId,
-                            proposedBoxNumberAddressPersistentLocalId)),
```

Comment:
I'm assuming that the facts deletions in this file and other test file is not because of refactoring but because of the business rule in which we don't reject/retire if it's not a destination? Is my assumption correct?

Reply:
Yes, we didn't change any test file in our refactor commit